src/styles/components/toc.scss

```diff
@@ -8,6 +8,8 @@ $navbar-height: 5.25rem; // 84px
   max-height: calc(100vh - 230px);
   overflow: auto;
   padding-right: 1rem;
+  animation-name: easeIn;
```

Comment:
You can use the `animation` shorthand property to replace these two properties.

Reply:
Got it 👍